primitives/src/currency.rs

```diff
@@ -138,6 +138,7 @@ create_currency_id! {
 		// Reserve for POLKABTC = 134
 		SDN("Shiden", 18) = 135,
 		// Reserve for PHA = 136
+		KILT("Kilt", 15) = 137,
```

Comment:
Is KILT going to be a Kusama parachain or Polkadot parachain? Make it 10 for Polkadot parachain

Reply:
we are going to be a Kusama chain for now.